### PR TITLE
Init vue cli

### PR DESCRIPTION
**Pull request by sweetcuptea**
Initializing vue CLI for the traxercise project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" href="/favicon.ico" />
```
Comment (trihargianto) [inline index.html:5]: where is the favicon file?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (trihargianto) [inline index.html:1]: do you modify this file? or this file is just a file from Vue cli?

File: package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "traxercise",
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview --port 4173",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs --fix --ignore-path .gitignore"
+  },
+  "dependencies": {
```
Comment (trihargianto) [inline package.json:10]: please remove the unnecessary dependencies and devDependencies. There are three kind of datepicker libraries. Please review all of the libraries you need

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (trihargianto) [inline .gitignore:1]: is that all? only node_modules?

File: vite.config.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+      "@": fileURLToPath(new URL("./src", import.meta.url)),
+    },
+  },
+  css: {
+    preprocessorOptions: {
+      scss: {
+        additionalData: `
+          @import "./src/assets/scss/fonts.scss";
+          @import "./src/assets/scss/variable.scss";
+          @import "./src/assets/scss/reset.scss";
```
Comment (trihargianto) [inline vite.config.js:35]: where is these files coming from?

File: vite.config.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+export default defineConfig({
+  plugins: [
+    vue2(),
+    vue2Jsx(),
+    legacy({
+      targets: ["ie >= 11"],
+      additionalLegacyPolyfills: ["regenerator-runtime/runtime"],
+    }),
+    fonts({
+      dirs: ["./src/assets/fonts"],
```
Comment (trihargianto) [inline vite.config.js:21]: where is this file coming from?

File: index.html (hunk 1 above)
Comment (sweetcuptea) [reply index.html:5]: there's is no favicon, i forgot to delete this code

Comment (trihargianto) [reply index.html:5]: Don’t delete the code. But provide the icon. An icon is essential for an app

**Review by trihargianto — APPROVED** [review]

**Pull request merged**